coordinator/src/escrow_coordinator/mod.rs

```diff
@@ -65,7 +63,7 @@ impl EscrowCoordinator {
         Ok(())
     }
 
-    async fn parse(&self, content: &str) -> anyhow::Result<([u8; 32], TradeContract)> {
+    async fn parse_contract(&self, content: &str) -> anyhow::Result<([u8; 32], TradeContract)> {
```

Comment:
I think we can make short, non IO functions sync, async is probably just overhead here?

Reply:
Yeah, good catch, I'll change it.